### PR TITLE
Update API Titles and Descriptions

### PR DESCRIPTION
**Pull request by em-lyn**
Updated character count limit:
- Title from 30 to 50 characters
- Overview from 80 to 150 characters

### CODE REVIEW

File: catalogue/wrds/reference-data-and-schema-lookup/api.yml [hunk 1]
```diff
@@ -1,10 +1,7 @@
 title: Reference Data & Schema Lookup
-description: Retrieves table schemas and data listing all fields in the database.
-type: REST
```
Comment (velsietis) [inline catalogue/wrds/reference-data-and-schema-lookup/api.yml:3]: looks like type got removed by mistake

File: catalogue/ac3/account-control-auth/api.yml [hunk 2]
```diff
@@ -1,10 +1,10 @@
-title: Account Control
-description: Authenticate and authorise users
+title: User authentication and authorisation
+description: Get access to the AC3 identity provider to authenticate users as well and manage and check their access rights.
```
Comment (velsietis) [inline catalogue/ac3/account-control-auth/api.yml]:
```suggestion
description: Authenticate users with their AC3 identity and verify their role and permissions
```

File: catalogue/mpi/pdq/api.yml [hunk 3]
```diff
@@ -1,12 +1,11 @@
-title: Patient Demographics Query
-description: Query for patients identifiers matched across multiple systems 
+title: Patient demographic information search and match
+description: Search for and match patient-identifying information across multiple sustems, such as NHS numnber, sex, date of birth and address.
```
Comment (BobbyCGD) [inline catalogue/mpi/pdq/api.yml:2]: spelling

File: catalogue/wcrs/documents-advanced-search/api.yml [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
+description:   Perform an advanced search of WCRS to find patient documents using Apache Solr. Group and filter results by category to find documents more quickly.
+type: Solr
 owner: NWIS - Corporate Applications
 status: stable
 access: restricted
 overview: |
-  We use Apache Solr to prvoide advanced search against patient documents in the Welsh Care Record Service.
+  Perform an advanced search of WCRS to find patient documents using Apache Solr.
+  Group and filter results by category to find documents more quickly.
+  We use Apache Solr to provide advanced search against patient documents in the Welsh Care Record Service.
```
Comment (velsietis) [inline catalogue/wcrs/documents-advanced-search/api.yml:8]: repetition (also as discussed "Apache Solr faceted search" could be considered a key word/phrase for describing the API)

**Review by velsietis — APPROVED** [review]

**Pull request merged**